bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -193,6 +225,10 @@
 		<item-type>Switch</item-type>
 		<label>Super Freeze</label>
 		<description>Start or stop Super Freezing</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
This is similar to Super Cool, I don't think it can be considered a mode.

Reply:
Changing to `Switch.Enabled`